src/py42/sdk/queries/fileevents/v2/filters/__init__.py

```diff
@@ -0,0 +1,8 @@
+from py42.sdk.queries.fileevents.v2.filters.destination_filter import *
```

Comment:
We should import each filter class from each module individually here (and in v1), otherwise we're getting the internal stuff like `FileEventFilterStringField` and `Choices` classes imported, which just clogs up the namespace for users.

Reply:
For v2, what do you think of switching it so that we actually import all the modules (rather than the classes from the modules) and the classes can be renamed and accessed more similarly to how they appear in the file event json?  This would also make looking for filter values easier (imo) because an IDE could prompt all possible filters relating to the `destination` module.

ie - 
`from py24.sdk.queries.fileevents.v2.filters import *`
`filter = destination.Name  # destination.py module, class Name()`

Reply:
I like it